OMCompiler/SimulationRuntime/fmi/export/openmodelica/fmu2_model_interface.c.inc

```diff
@@ -776,10 +774,8 @@ fmi2Status fmi2GetReal(fmi2Component c, const fmi2ValueReference vr[], size_t nv
 
   if (comp->_need_update)
   {
-    if (modelInitializationMode == comp->state)
+    if (comp->state == modelInitializationMode)
```

Comment:
Why did you move the constant value to the rhs? It was on the lhs on purpose.

Reply:
Well the value we are interested in is `comp->state` and that it is in `modelInitializationMode`. So I like to read it first.
Why would you do it the other way around (besides personal preferences)? I can revert it, I don't really care, only while reading the code it was simpler to understand for me.

Reply:
Constants on the left hand side avoid errors if it unintentionally get assigned instead of compared, i.e. typing = instead of ==. For that reason it is good to just be consistent and put constant on the lhs.

Reply:
Okay, I'll revert that change